test/units/modules/cloud/amazon/test_aws_s3.py

```diff
@@ -24,13 +24,6 @@ def test_is_fakes3(self):
         actual = s3.is_fakes3("fakes3://bla.blubb")
         self.assertEqual(True, actual)
 
-    def test_is_walrus(self):
```

Comment:
There doesn't seem to be a reference in the commit or the PR as to why this is being removed. I'm ok with it being removed if there's a good reason

Reply:
Because the is_walrus function it's no longer a part of the module. The tests haven't been running when the module is changed because of the file name.